core-bundle/contao/dca/tl_page.php

```diff
@@ -683,7 +684,37 @@ static function ($value) {
 			),
 			'sql'                     => "int(10) unsigned NOT NULL default 0",
 			'relation'                => array('type'=>'hasOne', 'load'=>'lazy')
-		)
+		),
+		'enableCsp' => array
+		(
+			'inputType'               => 'checkbox',
+			'eval'                    => array('submitOnChange'=>true),
+			'sql'                     => array('type'=>'boolean', 'default'=>false),
+		),
+		'csp' => array
+		(
+			'inputType'               => 'text',
+			'eval'                    => array('decodeEntities'=>true),
+			'sql'                     => array('type'=>'text', 'notnull'=>false),
+		),
+		'cspLevel1Fallback' => array
```

Comment:
Why even bother? Every browser supports level 2 and if you need to support legacy stuff, then please, handle the stuff yourself but do not ask Contao to do it for you.

Reply:
I also prefer not having this option (i.e. disabling it in general). Nelmio enables this by default, but I think we should not enable it by default (or at all).

Reply:
Removed in 13715f0811